conn/physic/units.go

```diff
@@ -1014,6 +1576,8 @@ const (
 	tera  prefix = 12
 )
 
+// take a prefix rune and converts it to an int. For use with indexing directly
```

Comment:
// parseSIPrefix converts a S.I. prefix run into an int. 

I'm not sure the rest is actually useful.